lib/app.js

```diff
@@ -40,7 +38,13 @@ function app (opts) {
       .catch(reject);
   });
 }
-
+/*
```

Comment:
we shouldn't have instructions pointing to 3rd party online services. 
also this is not the best place to put this kind of documentation. 
It is out of the scope of this PR, but it's preferrable to add a separate markdown file to the repo and link it from here. And if this sort of operation is necessary to fixing mappings, these instructions should be turned into a script with native node.js implementation of the operations (e.g. why use a 3rd party website to output JSON?)

Reply:
so remove this comment and you can put the content into an issue to plan for adding the necessary support doc/scripts